Test/git-issues/git-issue-1564-3.dfy

```diff
@@ -1,6 +1,6 @@
-// RUN: %dafny /compile:0 /functionSyntax:3 /env:0 /dprint:- "%s" > "%t"
```

Comment:
Are we dropping `/env`?

Reply:
I've made `env:0` the default and there's no `--env` yet